test/integration/infrastructure/infrastructure_test.go

```diff
@@ -127,17 +131,15 @@ var _ = AfterSuite(func() {
 		mgrCancel()
 	}()
 
+	By("running cleanup actions")
+	framework.RunCleanupActions()
```

Comment:
Does it mean all namespaces generated during testing are only be cleaned at the end of testing?  Is it expected behavior?